packages/admin/admin-rte/src/core/translation/htmlToState.spec.ts

```diff
@@ -0,0 +1,351 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import { convertFromRaw, EditorState, RawDraftContentState } from "draft-js";
+
+import { IRteOptions } from "../Rte";
+import { htmlToState } from "./htmlToState";
+import { stateToHtml } from "./stateToHtml";
+
+describe("htmlToState", () => {
+    const options = {
+        supports: [
+            "bold",
+            "italic",
+            "sub",
+            "sup",
+            "header-one",
+            "header-two",
+            "header-three",
+            "header-four",
+            "header-five",
+            "header-six",
+            "strikethrough",
+            "ordered-list",
+            "unordered-list",
+            "history",
+            "link",
+            "links-remove",
+            "non-breaking-space",
+            "soft-hyphen",
+        ],
+        listLevelMax: 4,
+        customToolbarButtons: [],
+        draftJsProps: { spellCheck: true },
+        standardBlockType: "unstyled",
+        blocktypeMap: {
+            unstyled: {
+                label: {
+                    type: {},
+                    key: null,
+                    ref: null,
+                    props: { id: "comet.rte.controls.blockType.default", defaultMessage: "Default" },
+                    _owner: null,
+                    _store: {},
+                },
+                renderConfig: { element: { propTypes: {}, options: { name: "CometAdminRteBlockElement" } }, aliasedElements: ["p"] },
```

Comment:
This seems wrong. Label looks like a React Node. Did you copy the options from the browser console?

Reply:
Yes, i copied it from browser console. I look into it, probably don't need everything.

Reply:
Removed everything that was not relevant for passing the test: 7e5c676dbaf649a48676a6548e34182c61839c10